app/javascript/controllers/sidebar_controller.js

```diff
@@ -0,0 +1,8 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  toggle() {
+    var expanded = (this.element.ariaExpanded === 'true');
+    this.element.ariaExpanded = !expanded;    
```

Comment:
@YukioArie alterei essa logica de .sidebar-mini para o atributo aria-expanded. Achei uma opção legal para reduzir código.
https://tailwindcss.com/docs/hover-focus-and-other-states#aria-states
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

Commit:
https://github.com/douglara/woofed-crm/pull/73/commits/f237619b0695bc4a2017b130bcd6036ad791b139